### DISCUSSION TITLE
Esta dando erro na instalação / com Install

### DISCUSSION THREAD

**Discussion by EfraimTec** [RamonSilva20/mapos#1753] [Q&A]
Aparece as seguintes mensagem ///

Sua instalação está incompleta, por favor instale as dependências utilizando o composer. A pasta system não foi localizada.

![image](https://user-images.githubusercontent.com/103220731/162307374-8ea246cb-5d43-4c21-8179-83552e0cb9a2.png)

**Comment by Pr3d4dor** [comment]
@EfraimTec Você precisa instalar as dependencias do projeto com o composer

**Comment by cwmkthubconnect** [comment]
Tenho mesmo erro meu composer esta instalado quando acesso composer -V

**Reply by cwmkthubconnect** [reply]
Composer version 2.3.7 2022-06-06 16:43:28

**Comment by Pr3d4dor** [comment]
@gruposocialinsta Qual versão do PHP?

**Reply by cwmkthubconnect** [reply]
O que procura amigo?

**Reply by Pr3d4dor** [reply]
Tentar te ajudar a resolver o problema que vc comentou acima
